src/main/scala/mesosphere/marathon/storage/migration/Migration.scala

```diff
@@ -213,9 +213,6 @@ object Migration {
       StorageVersions(1, 5, 0, StorageVersion.StorageFormat.PERSISTENCE_STORE) -> { (migration) =>
         MigrationTo15(migration)
       },
-      StorageVersions(1, 5, 2, StorageVersion.StorageFormat.PERSISTENCE_STORE) -> { (migration) =>
-        new MigrationTo152(migration.instanceRepo)
-      },
```

Comment:
I was unsure about this step. If one migrates through to 1.8 it is ok to drop this migration. However, if the migration fails after 1.6 and the user switches to the 1.6 binary we have a problem. What about dropping all migrations up to 1.6 and enforce that Marathon 1.8 can be only upgraded from 1.6 onwards?

Reply:
I think that by removing existing migrations you're going to open the gates of hell. I'd not do it, instead I'd fix the migration.

Reply:
If we say we support only upgrades from 1.6 upwards it's fine. I'd remove all migrations below 1.7 then. @meichstedt what is our support here?